cmd/job-trigger-controller-manager/main.go

```diff
@@ -29,6 +39,8 @@ func gatherOptions() (*options, error) {
 
 	fs.BoolVar(&o.dryRun, "dry-run", true, "Whether to run the controller-manager with dry-run")
 	fs.StringVar(&o.namespace, "namespace", "ci", "In which namespace the operation will take place")
+	fs.StringVar(&o.prowConfigPath, "prow-config-path", "", "Path to the Prow configuration directory.")
```

Comment:
Can you just use `k8s.io/test-infra/prow/flagutil/config` instead?

Reply:
Yes, you are right, we could do that instead :+1: